client/src/Components/Common/Navbar/Menu.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from "react";
+import SVG from "react-inlinesvg";
+
+import { Wrapper, MenuItem } from "./Menu.style.js";
+
+// import ProfileIcon from "./../../../assets/profile-icon.svg";
+
+export default class Menu extends Component {
+  render() {
+    const { width, isLoggedIn } = this.props;
+    return (
+      <Wrapper width={width}>
+        {width < 770 && (
```

Comment:
instead of checking against the width in every place, lets create a helper function called `isMobile(width)`, and also create a constant for mobile width and use it everywhere necessary

in client/src/helpers.js
`const isMobile = (width) => width < MOBILE_WIDTH`

in client/src/constants/screenWidths.js
`const MOBILE_WIDTH = 770`